latiss/transmission_filter/README.md

```diff
@@ -0,0 +1,78 @@
+This directory contains filter transmission curves for the 65mm LATISS
+filters, with measurements and processing as described below.
+
+Data Processing
+---------------
+
+The data in this directory have been extracted from the spreadsheet of
+values provided along with the email description of how the scans were
+performed.  The spreadsheet is included here for completeness.  After
+converting the spreadsheet to a CSV file, columns H-J,L-U were
+extracted.  The two measurements in the SDSSu_65mm~empty band were
+averaged together.  No normalization using column K performed in this
+band, as those values were all consistent with unity, and similar data
+is not available for the other bands.
+
+No negative values were clipped in any band, although they were likely
+caused by random noise during measurement.  For consistency with other
```

Comment:
Whether noise or light leaks, I wonder if we should clip them. Seems like having negative values could be problematic and/or confusing, and while it might not be "right", I feel like it can't be the _wrong_ call, given they're literally unphysical values.

Reply:
Yeah, I agree, it's probably best to let code using the values deal with these, it's a single line to clip them all, so I think on balance this is indeed the best course here.